src/OpenLoco/src/Windows/ObjectSelectionWindow.cpp

```diff
@@ -636,7 +639,7 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
     }
 
     // 0x00473579
-    static void drawPreviewImage(ObjectHeader* header, Gfx::DrawingContext& drawingCtx, int16_t x, int16_t y, Object* objectPtr)
+    static void drawPreviewImage(const ObjectHeader* header, Gfx::DrawingContext& drawingCtx, int16_t x, int16_t y, Object* objectPtr)
```

Comment:
I would prefer we pass things by reference when it is never null, the null check should happen at the place it obtains the ObjectHeader.